### PR TITLE
Improve docs

### LINKED ISSUE #258: Documentation uses coreos/alpine-sh, which is out of date
coreos/alpine-sh hasn't been updated for a year as I write. It is at v3.2.3, which is 2 versions behind and, for me at least, not something I can make use of.

I doubt very much that this image is the responsibility of this project, but the fact remains that your examples point to it in multiple places.  

Could we go into how to use _other_ images in the documentation a little?  The `acbuild dependency` documentation is pretty vague on this point.  Does "Example.com/imagename" refer to an actual website on the network, http://example.com, or the name of a container in the Rkt image library? If I want to use https://hub.docker.com/_/alpine/, can I? How?

### PR DESCRIPTION
**Pull request by sanmai-NL**
Resolves #258.

### CODE REVIEW

File: Documentation/dependencies.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-between `acbuild begin <image>` and `acbuild dependency add <image>`.
+between `acbuild begin <image>` and `acbuild dependency add <image>`. There is a 
+practical difference between the two methods `dep add` and `acbuild begin`. 
+`dep add` will point to an image in the manifest, and not include it in any way, 
+and `acbuild begin` will modify an image, meaning the resulting output will 
+include the entire contents of the image.
 
 This document aims to provide a detailed explanation of image dependencies in
-AppC and how they can be manipulated with acbuild.
+AppC and how they can be manipulated with acbuild.
```
Comment (lucab) [inline Documentation/dependencies.md]: Spurious trailing whitespace.

**Comment by sanmai-NL** [comment]
@lucab: Ready to merge?

**Comment by sanmai-NL** [comment]
@dgonyeo: ready to merge?

**Comment by dgonyeo** [comment]
LGTM

**Pull request merged**